sections/image-with-text.liquid

```diff
@@ -15,87 +15,93 @@
 {%- endstyle -%}
 
 {%- liquid
-  assign scheme1 = settings.color_schemes | first
   assign fetch_priority = 'auto'
   if section.index == 1
     assign fetch_priority = 'high'
   endif
+  if section.settings.color_scheme != section.settings.section_color_scheme
+    assign add_color_classes = true
```

Comment:
The trouble with excluding the color classes entirely, rather than relying on the "fake" transparent effect of using the same color scheme classes is that is can't hide overlaps and shadows when needed. I know with the motion side-effects we want to need to rely on true transparency more though.

This one with the overlap layout I think we probably need to address.
<img src="https://screenshot.click/03-08-1j084-kouot.png" width="300" />

Some others involving shadows like this have always been more edge casey so I wouldn't push back too hard on these regressions if our primary use cases are behaving correctly.

<img src="https://screenshot.click/03-25-x8vn6-vgumm.png" width="300" />
<img src="https://screenshot.click/03-07-vpfhg-34rup.png" width="300" />